backend/apps/github/models/user.py

```
@@ -55,7 +75,13 @@ class Meta:
     )
 
     contributions_count = models.PositiveIntegerField(
-        verbose_name="Contributions count", default=0
+        verbose_name="Contributions count",
+        default=0
+    )
+
+    calculated_score = models.FloatField(
+        default=0,
+        help_text="Computed score based on contribution signals"
     )
```

Ruff (0.15.6)

[warning] Class field `calculated_score` is defined multiple times

Remove duplicate field definition for `calculated_score`

(PIE794)